CHANGELOG.md

```diff
@@ -6,97 +6,97 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
-## [1.9.0] - 2020-03-03
+## [1.9.0](https://github.com/algolia/scout-extended/releases/tag/v1.9.0) - 2020-03-03
```

Comment:
Thanks for this pull request, actually each line should be like so:
```
https://github.com/algolia/scout-extended/compare/v1.8.0...v1.9.0
```

Reply:
Thanks for reviewing!

Changed the links except for the v0.1.0 release as there's no release before that. Is there another way where you can compare from start to v0.1.0? 🙂